File: tests/fixtures/languages/sample.kt
// Minimal Kotlin fixture for language parse-validation tests.

package demo

import kotlin.collections.List

class Greeter(val label: String) {
    fun greet(): String {
        return "Hello, $label"
    }
}

object Util {
    fun add(a: Int, b: Int): Int = a + b
}

fun square(x: Int): Int = x * x

File: tests/fixtures/languages/sample.js
// Minimal JavaScript fixture for language parse-validation tests.

function add(a, b) {
    return a + b;
}

class Greeter {
    constructor(label) {
        this.label = label;
    }

    greet() {
        return `Hello, ${this.label}`;
    }
}

const square = (x) => x * x;
